torch/ao/sparsity/_experimental/data_sparsifier/lightning/tests/test_callbacks.py

```diff
@@ -0,0 +1,89 @@
+from torch.ao.sparsity._experimental.data_sparsifier.data_norm_sparsifier import DataNormSparsifier
+import torch
+import pytorch_lightning as pl  # type: ignore[import]
+import torch.nn as nn
+from typing import List
+from torch.ao.sparsity._experimental.data_sparsifier.lightning.callbacks.data_sparsity import PostTrainingDataSparsity
+from torch.ao.sparsity._experimental.data_sparsifier.lightning.callbacks._data_sparstity_utils import _get_valid_name
+from torch.ao.sparsity._experimental.data_sparsifier.base_data_sparsifier import SUPPORTED_TYPES
+from torch.testing._internal.common_utils import TestCase
+from torch.testing._internal.common_utils import run_tests
+
+
+class DummyModel(nn.Module):
+    def __init__(self, iC: int, oC: List[int]):
+        super().__init__()
+        self.linears = nn.Sequential()
+        self.emb = nn.Embedding(1024, 32)
+        self.emb_bag = nn.EmbeddingBag(1024, 32)
+        i = iC
+        for idx, c in enumerate(oC):
+            self.linears.append(nn.Linear(i, c, bias=False))
+            if idx < len(oC) - 1:
+                self.linears.append(nn.ReLU())
+            i = c
+
+
+class DummyLightningModule(pl.LightningModule):
+    def __init__(self, iC: int, oC: List[int]):
+        super().__init__()
+        self.model = DummyModel(iC, oC)
+
+    def forward(self):
+        pass
```

Comment:
Let's wrap this into a creator function to encapsulate the pl import? Something like:

```
def make_lightning_module(*args, **kwargs):
    import pytorch_lightning as pl # type: ignore[import]
    class DummyLightningModule(pl.LightningModule):
        ...
    return DummyLightningModule(*args, **kwargs)
```

Reply:
yep